win2xcur/cursor.py

```diff
@@ -16,6 +16,10 @@ def __init__(self, image: SingleImage, hotspot: Tuple[int, int], nominal: int) -
     def __repr__(self) -> str:
         return f'CursorImage(image={self.image!r}, hotspot={self.hotspot!r}, nominal={self.nominal!r})'
 
+    def scale(self, width: int, height: int) -> None:
+        self.image.scale(width, height)
+        self.nominal = max(width, height)
```

Comment:
This makes sense when generating new sizes, but I am not yet convinced it actually makes sense in general.

The nominal size is a special field in the Xcursor format that helps in selecting the image to use, but the actual image can be any size you want. There are weird situations with cursors that have a lot of padding around the actual image. In those cases, the outcome might not be desirable, which is why scaling was originally introduced. If we adjust the nominal size, then on the Linux side, a different cursor sprite would be selected, resulting in identical visual outcomes.